### ISSUE TITLE
Unable to resize Ellipses both in full debrief and  D-Lite

### ISSUE THREAD

**Issue by helenayele** [debrief/debrief#4221]
It is not possible to resize ellipse shape using Drag Element/Component feature of both D-Lite and full debrief.

### Steps to reproduce
1. Launch D-Lite 
2. Insert Ellipse shape in any layer  
3. On View menu, first select Fit to window button and  click Drag Element button 
4. Then mouse over on the ellipse shape
5. When the cursor hand turns to green try to resize it
See that it will not resize it
[![Image from Gyazo](https://i.gyazo.com/580e5f0cb444594f71aac50ca5fb2707.gif)](https://gyazo.com/580e5f0cb444594f71aac50ca5fb2707)

The same is true for full debrief also , the cursor hand never turns to green
[![Image from Gyazo](https://i.gyazo.com/f73c0cc2de51a5b5cabee5b98843588d.gif)](https://gyazo.com/f73c0cc2de51a5b5cabee5b98843588d)

**Comment by ayeshare** [comment]
I spent some time on this issue. Found that EllipseShape.java does not implement HasDraggableComponents while all other shape objects implement HasDraggableComponents.
CircleShape.java implements methods shift() and findNearestHotSpotIn. I compared this with Circle and found that for this shape the drag is based on the point Radius.  In ellipse it has to be Maxima and/or Minima.  When I tried replacing the circle's radius with similar code in maxima of ellipse, i could drag it around Maxima.
